From: John Paul Adrian Glaubitz <glaubitz@physik.fu-berlin.de>
Date: Mon, 12 Mar 2012 01:17:08 +0100
Subject: Fix type declaration for gzf in struct dsk_data_t. gzFile from
 zlib.h is already of type pointer, so no need to declare gzf as
 gzFile *.

---
 src/libdisk/load_22dsk.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libdisk/load_22dsk.c b/src/libdisk/load_22dsk.c
index 5c1309d..3f52053 100644
--- a/src/libdisk/load_22dsk.c
+++ b/src/libdisk/load_22dsk.c
@@ -81,7 +81,7 @@
 
 typedef struct {
   FILE *f;
-  gzFile *gzf;
+  gzFile gzf;
   int read_only;
   int len;
   struct {
